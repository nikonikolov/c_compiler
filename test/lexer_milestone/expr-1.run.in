
int f()
{
int x=0+0;
int u=x-x;
int z= u  * u; 
int y= z / x;
 return y;
}
