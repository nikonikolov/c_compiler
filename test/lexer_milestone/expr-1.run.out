int Keyword Tint 2  2
f Identifier TIdentifier 2  2
( Operator T( 2  2
) Operator T) 2  2
{ Operator T{ 3  3
int Keyword Tint 4  4
x Identifier TIdentifier 4  4
= Operator T= 4  4
0 Constant TIntConstant 4  4
+ Operator T+ 4  4
0 Constant TIntConstant 4  4
; Operator T; 4  4
int Keyword Tint 5  5
u Identifier TIdentifier 5  5
= Operator T= 5  5
x Identifier TIdentifier 5  5
- Operator T- 5  5
x Identifier TIdentifier 5  5
; Operator T; 5  5
int Keyword Tint 6  6
z Identifier TIdentifier 6  6
= Operator T= 6  6
u Identifier TIdentifier 6  6
* Operator T* 6  6
u Identifier TIdentifier 6  6
; Operator T; 6  6
int Keyword Tint 7  7
y Identifier TIdentifier 7  7
= Operator T= 7  7
z Identifier TIdentifier 7  7
/ Operator T/ 7  7
x Identifier TIdentifier 7  7
; Operator T; 7  7
return Keyword Treturn 8  8
y Identifier TIdentifier 8  8
; Operator T; 8  8
} Operator T} 9  9
